eng/RTMVersions/RTMVersions.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!--
+    Gather project references for compilation against RTM packages. %(RTMVersion) is set for about a dozen packages
+    in all servicing builds. Cannot reference two versions of a package, mandating this separate package.
+  -->
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Can't be parameterized?

Reply:
Not unless I remove the `Directory.Build.*` files. I'll test out whether that causes any problems after getting this in and doing the merge to 'master'. It's no worse at the moment than the hard-coded TFMs in (say) our project templates. But, I'll clean it up next if possible.